content-scripts/cs.js

```diff
@@ -17,6 +17,25 @@ if (pathArr[0] === "scratch-addons-extension") {
 }
 if (path === "discuss/3/topic/add/") {
   window.addEventListener("load", () => forumWarning("forumWarning"));
+  let uaElemModified = false;
+  const modifyUAElem = () => {
+    if (uaElemModified) return;
+    const uaElem = document.getElementById("simple-user-agent");
+    if (uaElem) {
+      uaElem.textContent = uaElem.textContent.replace("/", "and");
```

Comment:
Dunno, but how about removing the surrounding spaces? I got bugged with that.
```js
uaElem.textContent = uaElem.textContent.replace(" / ", "/")
```

Reply:
i dont want it to be too suttle

Reply:
Can't we add an invisible character in a specific place? Then make cs.js or module.js detect those and put a Scratch Addons logo or something next to the user agent with a hover title that says "this is a Scratch Addons user"